packages/contracts-bedrock/scripts/merkle-trie-fuzzer/README.md

```diff
@@ -0,0 +1,3 @@
+# `merkle-trie-fuzzer`
```

Comment:
Perhaps we can make this not specific to merkle tree fuzzing and expand it over time to include all of the different fuzz targets by calling it `fuzzer` or something more general